modules/Data/FileFormat.yaml

```diff
@@ -102,6 +102,15 @@ enums:
       cns:
         description: Segmented log2 ratios from CNVKit pipeline, these represent the most relevant output describing copy number variation. 
         source: https://cnvkit.readthedocs.io/en/stable/fileformats.html
```

Comment:
@anngvu what is the difference between "meaning" and "source"? Should I be using one or the other?

Reply:
`meaning` should be an ontology uri. `source` is for some reference (so "Sage Bionetworks", maybe even "James Eddy", but most commonly some website or paper)

Reply:
I'll add this topic to the much-needed doc update...

Reply:
Ok! I'll change this to "source" where relevant.

Reply:
So I don't think I completely segregated our original `source` values for terms into LinkML's particular of `meaning` vs `source` when I did the migration to LinkML, so this isn't too important for now. I think we can go back and clean this up when we do need to make sure that `meaning` are valid uris.